client/src/components/Donation/donation-text-components.tsx

```diff
@@ -53,3 +84,130 @@ export const DonationOptionsAlertText = (): JSX.Element => {
     </p>
   );
 };
+
+const FaqItem = (
+  title: string,
+  text: JSX.Element,
+  key: number
+): JSX.Element => {
+  const [isExpanded, setExpanded] = useState(false);
+  return (
+    <div className={`faq-item ${isExpanded ? 'open' : ''}`} key={key}>
+      <button className='map-title' onClick={() => setExpanded(!isExpanded)}>
+        <Caret />
+        <h4>
+          <b>{title}</b>
+        </h4>
+      </button>
+      {isExpanded && (
+        <>
+          <div className='map-challenges-ul'>{text}</div>
+        </>
+      )}
+    </div>
+  );
+};
+
+export const DonationFaqText = (): JSX.Element => {
+  const { t } = useTranslation();
+  const faqItems = [
+    { Q: t('donate.get-help'), A: <p>{t('donate.forward-receipt')}</p> },
+    {
+      Q: t('donate.how-transparent'),
+      A: (
+        <>
+          <p>{t('donate.very-transparent')}</p>
+          <p>
+            <Trans i18nKey='donate.download-irs'>
+              <a href={t('links:donate.download-irs-url')}>placeholder</a>
+            </Trans>
+          </p>
+          <p>
+            <Trans i18nKey='donate.download-990'>
+              <a href={t('links:donate.download-990-url')}>placeholder</a>
+            </Trans>
+          </p>
+        </>
+      )
+    },
+    {
+      Q: t('donate.how-efficient'),
+      A: (
+        <>
+          <p>{t('donate.fcc-budget')}</p>
+          <p>{t('donate.help-millions')}</p>
+        </>
+      )
+    },
+    {
+      Q: t('donate.how-one-time'),
+      A: (
+        <>
+          <p>
+            <Trans i18nKey='donate.one-time'>
+              <a href={t('links:donate.one-time-url')}>placeholder</a>
+            </Trans>
+          </p>
+          <p>{t('donate.wire-transfer')}</p>
+        </>
+      )
+    },
+    {
+      Q: t('donate.does-crypto'),
+      A: (
+        <>
+          <p>{t('donate.yes-crypto')}</p>
+          <p>{WALLETS}</p>
+        </>
+      )
+    },
+
+    {
+      Q: t('donate.can-check'),
+      A: (
+        <>
+          <p>{t('donate.yes-check')}</p>
+          <p>{POBOX}</p>
+        </>
+      )
+    },
+    {
+      Q: t('donate.how-matching-gift'),
+      A: (
+        <>
+          <p>{t('donate.employers-vary')}</p>
+          <p>{t('donate.some-volunteer')}</p>
+          <p>{t('donate.help-matching-gift')}</p>
+        </>
+      )
+    },
+    { Q: t('donate.how-endowment'), A: <p>{t('donate.endowment')}</p> },
+    {
+      Q: t('donate.how-legacy'),
+      A: (
+        <>
+          <p>{t('donate.we-honored')}</p>
+          <p>
+            <blockquote>{t('donate.legacy-gift-message')}</blockquote>
+          </p>
+          <p>{t('donate.thank-wikimedia')}</p>
```

Comment:
Error:
```
Warning: validateDOMNesting(...): <blockquote> cannot appear as a descendant of <p>.
    in blockquote (created by DonationFaqText)
    in p (created by DonationFaqText)
    in div (created by DonationFaqText)
    in div (created by DonationFaqText)
    in DonationFaqText (created by DonatePage)
```

```suggestion
          <p>{t('donate.we-honored')}</p>
          <blockquote cite="https://commons.wikimedia.org/wiki/Main_Page">
              <p>{t('donate.legacy-gift-message')}</p>
          </blockquote>
          <p>{t('donate.thank-wikimedia')}</p>
```

If you do not like the `cite`, just edit - it is not entirely necessary